src/api/hooks/useGetMainnetValidatorsGeoData.ts

```diff
@@ -40,7 +40,7 @@ export function useGetValidatorSetGeoData() {
     const groups: ValidatorGeoGroup[] = validators.reduce(
       (groups: ValidatorGeoGroup[], validatorData: MainnetValidatorData) => {
         const geoData = validatorData.location_stats;
-        const country = geoData.country;
+        const country = geoData?.country;
```

Comment:
I suggest changing this type https://github.com/aptos-labs/explorer/blob/61cd024c7e4bfd01a452abd1badfa10420f0b606/src/api/hooks/useGetMainnetValidators.ts#L17 to reflect the fact that it can be `undefined`.